frontend/src/views/Home.vue

```diff
@@ -14,6 +14,7 @@
 import HomePageButtons from "../components/HomePageButtons";
 export default {
   name: "Home",
+  title: "Home",
```

Comment:
For home page there is no need in this. Let it be just “Audio Seva”. Possible?

Reply:
I think vue-page-title does not support a change like that. I think because Home is the exception a solution might be to change it through `document.title` on the Home page